### ISSUE TITLE
Add an option to show images by converting image sources to Base64

### ISSUE THREAD

**Issue by emrecoban** [lukeaschenbrenner/TxtNet-Browser#13]
I noticed that the images in the app are not being rendered in the browser. I understand that this can be costly, but it would be great to have an optional "show images" feature. To achieve this, it would be sufficient to convert all image elements in the source code to Base64 format.

_I suggest adding an "improvement" tag to this issue. Thank you for considering this feature._

**Comment by lukeaschenbrenner** [comment]
Interesting idea. I think we would want to do some pre-processing on the images to specify a max size and use compression to get there too. Also, this would need to be opt-in at both the client and server level, as I would imagine sending images over SMS will be quite costly. Initial thoughts are that a server has a global setting to allow images, then the image URLs are included in the image tags. When a user taps on the image, they send a request for the image URL back to the server. The server will then download and compress the image, and when it is sent back, gets placed into the html as base64. I will look into this, thanks for the suggestion!

**Comment by emrecoban** [comment]
Yes! I know this process will take a long time, but it will solve the issue.
